reconcile/test/test_terraform_repo.py

```diff
@@ -209,6 +301,62 @@ def test_update_repo_state(int_params, existing_repo, state_mock):
     )
 
 
+# these two output tests are to ensure that there isn't a sudden change to outputs that throws
+# off tf-executor
+def test_output_correct_statefile(
+    int_params_print_to_tmp, existing_repo, existing_repo_output, tmp_path, state_mock
+):
+    integration = TerraformRepoIntegration(params=int_params_print_to_tmp)
+
+    existing_state: list = []
+    desired_state = [existing_repo]
+
+    expected_output = yaml.safe_load(existing_repo_output)
+
+    diff = integration.calculate_diff(
+        existing_state=existing_state,
+        desired_state=desired_state,
+        dry_run=True,
+        state=state_mock,
+    )
+
+    assert diff
+    if diff:
```

Comment:
`diff` is already `assert`, why need to `if` check again?

Reply:
mypy was complaining about `if` possibly being None however doesn't seem to be giving me issues now